src/kansei_main.cpp

```diff
@@ -46,23 +46,23 @@ int main(int argc, char * argv[])
     "Optional:\n"
     "-h, --help           show this help message and exit\n";
 
-  if (argc > 1) {
-    for (int i = 1; i < argc; i++) {
-      std::string arg = argv[i];
+  if (args.size() > 1) {
+    for (int i = 1; i < args.size(); i++) {
+      std::string arg = args[i];
```

Comment:
Should use a constant reference to prevent variable copy:
```suggestion
      const std::string& arg = args[i];
```